iosApp/iosApp/ui/pages/ForgotPasswordScreen.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+import Shared
+import KMMViewModelSwiftUI
+
+struct ForgotPasswordScreen: View {
+    @ObservedObject var navigation: NavigationController
+    @StateViewModel var viewModel = ForgotPasswordViewModel(firebaseRepo: FirebaseRepository())
+    let strings = StringsHelper().getResourceStrings()
+    
+    var body: some View {
+        HStack {
+            Button(action: {
+                navigation.navigateBack()
```

Comment:
Isn't it possible to use the same action/event based way as android?

Reply:
In this case the onAction(ForgotPasswordAction.OnNavigateBackClicked) and fetching the event to navigate back